lang/cs/Org.Apache.REEF.Wake/Avro/ProtocolSerializer.cs

```diff
@@ -36,36 +36,52 @@ public sealed class ProtocolSerializer
     {
         private static readonly Logger Logr = Logger.GetLogger(typeof(ProtocolSerializer));
 
-        // Delagates for message serializers and deserializers.
+        /// <summary>
+        /// Delegate for message serializer.
+        /// </summary>
         private delegate void Serialize(MemoryStream stream, object message);
+
```

Comment:
By default, doxygen does not put private and protected member documentation in the output.